Casks/itchysats.rb

```diff
@@ -0,0 +1,11 @@
+cask "itchysats" do
+  version "0.8.0"
+  sha256 "07866b1334d7049599958e1181de4e9b4b3ce82f43a2829bbef3817c009fa89e"
+
```

Comment:
🤔 I think we are going to publish a dmg for intel and M1 chips. Can you add a check to download the correct dmg?

Reply:
I think the best way forward is to publish an universal binary (single dmg for both chips), which is a de-facto standard nowadays.

Reply:
I've opened a separate tracking ticket for native ARM #2